src/bezpath.rs

```diff
@@ -1108,10 +1108,10 @@ impl From<QuadBez> for PathSeg {
 }
 
 impl Shape for BezPath {
-    type PathElementsIter = std::vec::IntoIter<PathEl>;
+    type PathElementsIter<'iter> = std::iter::Copied<std::slice::Iter<'iter, PathEl>>;
 
-    fn path_elements(&self, _tolerance: f64) -> Self::PathElementsIter {
-        self.0.clone().into_iter()
```

Comment:
This is the main benefit from this PR - now we don't have to clone the whole object.